shared/src/gameData/characterData.ts

```diff
@@ -2252,7 +2210,6 @@ export const CHARACTERS: CharacterDefinition[] = [
     otherNight: {
       reminder: "The Lleech points to a player. That player dies.",
       order: 34,
-      status: [{ type: "dead" }],
```

Comment:
I don't love getting rid of all of this data without replacing it with other structured data.